tests/drivers/entropy/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+MESSAGE("Current dir:" ${CMAKE_CURRENT_SOURCE_DIR})
+set(app_sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/main.c
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../../drivers/entropy/entropy_ll_nrfxlib.c
+  )
+
+set(OPTIONS
+    -DUNIT_TEST
+    -DNRF_IRQ_RNG_IRQn=5
```

Comment:
What is it supposed to be used for?

Reply:
I suspect it was there for the code to compile originally, there has been some changes upstream since this was written, so I can remove it.